events/RDS/StartExportTask.json

```diff
@@ -9,6 +9,35 @@
     "mitreAttackTechniques": [
         "T1537 - Transfer Data to Cloud Account"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1567 - Exfiltration Over Web Service",
+            "reason": "By exporting data to an S3 bucket, adversaries can use cloud services as a method to exfiltrate data without direct interaction with the database."
+        },
+        {
+            "technique": "T1530 - Data from Cloud Storage Object",
+            "reason": "After exporting snapshot data to S3, an adversary can retrieve and analyze the data from the S3 bucket, provided they maintain access to the cloud storage."
+        },
+        {
+            "technique": "T1552 - Unsecured Credentials",
+            "reason": "If the database snapshots contain unsecured credentials, such as plaintext passwords or API keys, exporting the snapshot to S3 could allow an adversary to access these credentials, which could then be used to further compromise the environment or exfiltrate additional data."
```

Comment:
I agree but I don't see the StartExportTask extrictly related to unsecured credentials.

Reply:
Removed.